### ISSUE TITLE
Webpacker::Env.current cannot be changed when using NODE_ENV and spring 

### ISSUE THREAD

**Issue by activefx** [rails/webpacker#205]
Webpacker::Env.current cannot be changed with NODE_ENV after initial load when using spring (unless spring is disabled or stopped). 

### Steps to reproduce

Using webpacker master branch (3564a1c) with Rails 5.1.0.rc1 

### Expected behavior

    RAILS_ENV=development rake webpacker:compile
    #=> Webpacker::Env.current
    #=> "development"

    NODE_ENV=test rake webpacker:compile
    #=> Webpacker::Env.current
    #=> "test"

### Actual behavior

    RAILS_ENV=development rake webpacker:compile
    #=> Webpacker::Env.current
    #=> "development"

    NODE_ENV=test rake webpacker:compile
    #=> Webpacker::Env.current
    #=> "development"

### System configuration
**Rails version**:
5.1.0.rc1
**Ruby version**:
ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-darwin15]
**Webpacker**:
3564a1cb13d5

**Comment by mfazekas** [comment]
Can't you just use `RAILS_ENV=test rails webpacker:compile` or `RAILS_ENV=test rake webpacker:compile`? I assume those are working.

**Comment by gauravtiwari** [comment]
@activefx Yes as @mfazekas pointed out please use `RAILS_ENV` as it's refreshed on the fly. You can read more here - https://github.com/rails/spring#commands and . The other option is to use something like `figaro` or `dotenv` and have spring watch them for changes - https://github.com/rails/spring#watching-files-and-directories

```ruby
# inside an initializer
Spring.watch 'config/application.yml'
```

The last option would be to restart spring if you change `NODE_ENV`

**Comment by gauravtiwari** [comment]
Perhaps, we could do this too?  

```ruby
if defined?(Spring)
  Spring.after_fork do
    # Reload 
    Webpacker::Env.load
  end
end
```

**Comment by dhh** [comment]
Closed by #221
